Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
Translators are useful, but hardly an academically valid resource.
"Quieres" may be understood, but it's a matter of quality writing.

Please do take a look at the resources I referred you to.
It's a terrible idea to use ONLY google translate. It translates chunks of text, but is no authority for "reglas de redacción".

Are you a native Spanish speaker, may I ask?